README.md

```diff
@@ -1,24 +1,32 @@
-                               Autopsy 3.0
-                          http://www.sleuthkit.org/
+#                               Autopsy 4.0
+http://www.sleuthkit.org/
 
-                                March 13, 2013
+March 3, 2016
 
 
-OVERVIEW
+
+##OVERVIEW
 
 Autopsy is a graphical interface to The Sleuth Kit and other open source digital forensics tools. 
-Autopsy 3 is a complete rewrite from Autopsy 2 and it is now Java-based.     
+Autopsy 4 (and 3) are a complete rewrite from Autopsy 2, and none of this document is relevant to Autopsy 2.   
 
 Although Autopsy is designed to be cross-platform (Windows, Linux, MacOSX), the current version is fully functional and fully tested only on Windows. 
-We have run it on XP, Vista, and Windows 7 with no problems. 
+You can run Autopsy 4 on Linux and OS X, but it must be built from source code. 
```

Comment:
Why delete the statement that we have run Autopsy on Windows XP, Vista, and Windows 7? Note that as far as I know, no one here at Basis Technology has tried to run Autopsy on Windows 8 or 10.

Autopsy is intended to be cross-platform, but have you built Autopsy for Linux and/or OS-X and run it successfully? I have heard reports of Linux builds and we are planning to attempt an OS-X build in the near future, so I would be very interested in hearing about your experiences if you have done these builds. If not, we should not make any stronger claims about Linux and OS-X builds until it is verified that they can be done, at which time we may wish to augment this document with Linux and OS-X build details.